director-configure-blobstore.html.md.erb

```diff
@@ -72,3 +72,49 @@ The Director and the Agents can use an S3 compatible blobstore. Here is how to c
         bucket_name: test-bosh-bucket
         host: objects.dreamhost.com
     ```
+
+---
+## <a id="gcs"></a> Google Cloud Storage (GCS)
+
+The Director and the Agents can use GCS blobstore. Here is how to configure it:
+
+1. Create a GCS bucket.
+
+1. Ensure that access to the bucket is protected, as the Director may store sensitive information.
+
+1. Modify deployment manifest for the Director and specify GCS credentials and bucket name:
+
+    ```yaml
+    properties:
+      blobstore:
+        provider: gcs
+        json_key: |
+          SERVICE-ACCOUNT-FILE
+        bucket_name: test-bosh-bucket
+    ```
+
+1. To use [Customer Supplied Encryption Keys](https://cloud.google.com/storage/docs/encryption#customer-supplied)
+to encrypt blobstore contents instead of server-side encryption keys, specify `encryption_key`:
+
+    ```yaml
+    properties:
+      blobstore:
+        provider: gcs
+        json_key: |
+          SERVICE-ACCOUNT-FILE
+        bucket_name: test-bosh-bucket
+        encryption_key: AAECAwQFBgcICQoLDA0ODxAREhMUFRYXGBkaGxwdHh8=
```

Comment:
people love to copy and paste so better not give them a valid key here. maybe 'REPLACE-ME-WITH-[N?]-CHARACTER-STRING`